Lib/test/datetimetester.py

```diff
@@ -2985,7 +2991,7 @@ def test_strptime_leap_year(self):
             self.theclass.strptime('02-29,2024', '%m-%d,%Y')
 
     def test_strptime_z_empty(self):
-        for directive in ('z',):
+        for directive in ('z', ':z'):
```

Comment:
Was it foreshadowing that this was already looping over a single directive? Kinda funny. 😛